public/index.html

```diff
@@ -14,6 +14,20 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>QPP Submissions API Developer Documentation</title>
+    <!--START Google Analytics tracking code - to set up cross-domain tracking from qpp.cms.gov to cmsgov.github.io-->
+    <script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+      
+  ga('create', 'UA-15356370-63', 'auto', {'allowLinker': true});
+  ga('require', 'linker');
+  ga('linker:autoLink', ['qpp.cms.gov'] );
+  ga('send', 'pageview');
+
+</script>
```

Comment:
oh - one last nitpick, the indentation here should match the opening tag, and the contents should be indented two spaces in from the script tags